plugins/catalog-backend-module-ldap/config.d.ts

```diff
@@ -17,8 +17,223 @@
 import { JsonValue } from '@backstage/types';
 
 export interface Config {
+  /**
+   * LdapOrgEntityProvider / LdapOrgReaderProcessor configuration
+   */
+  ldap?: {
```

Comment:
this is a verbatim copy of the old block below which is WAY overdue for deprecation but let's do that separately; this addition just reflects the current reality which used to misrepresented